FF1Lib/AltFiends.cs

```diff
@@ -3275,23 +3275,27 @@ public void AlternativeFiends(EnemyScripts enemyScripts, MT19337 rng, Flags flag
 				Put(EnemyOffset + (FiendsIndex + i) * EnemySize, fiends[i].compressData());
 			}
 
-			//Update fiends names, we stack Fiend1 and Fiend2's names to get more space for names
-			var enemyText = ReadText(EnemyTextPointerOffset, EnemyTextPointerBase, EnemyCount);
+			
+			// var enemyText = ReadText(EnemyTextPointerOffset, EnemyTextPointerBase, EnemyCount);
```

Comment:
so sometime you might want to just comment out code because you might want to restore it later, but in general it should be out right deleted, no need to keep it around